### PR TITLE
Fix bug in hydrostatic spherical Coriolis

### LINKED ISSUE #2929: Bugfix in hydrostatic spherical coriolis
https://github.com/CliMA/Oceananigans.jl/blob/c929676bcce7fc19c3f96faafd459371935a6eba/src/Coriolis/hydrostatic_spherical_coriolis.jl#L63

should be

```
neighbouring_active_nodes = @inbounds ℑxyᶠᶜᵃ(i, j, k, grid, revert_peripheral_node, Center(), Face(), Center()) 
```

(and so on for the other instances of the function)

### PR DESCRIPTION
**Pull request by navidcy**
Closes #2929

### CODE REVIEW

**Review by glwagner — APPROVED** [review]

**Comment by navidcy** [comment]
@simone-silvestri I'd be keen to hear your input on the Pr before merging.

**Review by simone-silvestri — APPROVED** [review]

**Comment by navidcy** [comment]
@simone-silvestri, shall I bump up the release? 
I'm not sure what the bug affects...

File: src/Coriolis/hydrostatic_spherical_coriolis.jl [hunk 1]
```diff
@@ -60,13 +60,13 @@ const CoriolisActiveCellEnstrophyConserving = HydrostaticSphericalCoriolis{<:Act
 @inline revert_peripheral_node(args...) = !peripheral_node(args...)
```
Comment (glwagner) [inline src/Coriolis/hydrostatic_spherical_coriolis.jl:60]:
```suggestion
@inline not_peripheral_node(args...) = !peripheral_node(args...)
```

Comment (glwagner) [reply src/Coriolis/hydrostatic_spherical_coriolis.jl:60]: Bring sanity to the world

**Comment by simone-silvestri** [comment]
It affected only the use of `ActiveCellEnstrophyConservingScheme`. 
Maybe it is good to bump although I don't think many users were using that option

**Pull request merged**